slasher/rpc/server.go

```diff
@@ -91,5 +91,37 @@ func (ss *Server) IsSlashableAttestation(ctx context.Context, req *ethpb.Indexed
 // IsSlashableBlock returns an proposer slashing if the block submitted
 // is a double proposal.
 func (ss *Server) IsSlashableBlock(ctx context.Context, req *ethpb.SignedBeaconBlockHeader) (*slashpb.ProposerSlashingResponse, error) {
-	return nil, errors.New("unimplemented")
+	ctx, span := trace.StartSpan(ctx, "detection.IsSlashableBlock")
+	defer span.End()
+
+	gvr, err := ss.beaconClient.GenesisValidatorsRoot(ctx)
+	if err != nil {
+		return nil, err
+	}
+	blockEpoch := helpers.SlotToEpoch(req.Header.Slot)
```

Comment:
Can we make slasher panic if we pass in a bad request? Maybe add a nil check

Reply:
added. thanks